app.yaml

```diff
@@ -1,6 +1,12 @@
 runtime: nodejs14
 entrypoint: node -r esm src/server.js
 instance_class: F1
+
 automatic_scaling:
   max_instances: 1
   min_instances: 0
+
+env_variables:
+  NODE_ENV: $NODE_ENV
```

Comment:
google app engine sets this to production automatically https://cloud.google.com/appengine/docs/standard/nodejs/runtime#environment_variables

Reply:
👍